### ISSUE TITLE
Interactive console for "Running fixes" usability improvements

### ISSUE THREAD

**Issue by m-kuhn** [sqlfluff/sqlfluff#638]
Running sqlfluff with fixes activated gives an interactive prompt if you want to attempt to fix.
The prompt suggests a default selection of Yes by capitalizing the `Y` in `[Y/n]`. However after hitting "enter", the default is not respected and an invalid input is reported.

```py
sqlfluff fix file.sql --rules [someruleshere]
==== finding fixable violations ====
== [file.sql] FAIL
...
==== fixing violations ====
[some] fixable linting violations found
Are you sure you wish to attempt to fix these? [Y/n] ... # Accept the "default" selection "Y" by hitting "Enter"
Invalid input :(
Aborting...
```

The message itself "Invalid input" can also be misleading and raise the assumption that it's the input file and not the keystroke which is invalid. (I started to bisect the error inside the file by shortening it down before realizing what the error was actually supposed to mean).
